include/roboteam_ai/interface/widgets/widget.h

```diff
@@ -41,25 +38,26 @@ class Visualizer : public QWidget {
     void setShowRobotInvalids(bool showPath);
     void setShowBallPlacementMarker(bool showMarker);
     void setShowDebugValueInTerminal(bool showDebug);
-    void toggleSelectedRobot(int robotId);
+    void toggleSelectedRobot(rtt::world_new::view::RobotView robot);
     void setToggleFieldDirection(bool inversed);
 
    protected:
     void paintEvent(QPaintEvent *event) override;
     void mousePressEvent(QMouseEvent *event) override;
 
    private:
-    float factor;
+    const rtt::world_new::World &worldManager;
+    float factor{};
```

Comment:
wtf is this bracket initialization? Have never seen this before.

Reply:
Ah, clang did that for me. It was complaining that it was uninitialized